brave/browser-laptop
# Adds confirmation dialog upon clicking the trash icon in the ledger table.

Fixes Issue #11164

## Submitter Checklist:


## Test Plan:

 1. Visit some sites
 2. Open about:payments
 3. Click a trash icon
 4. Ensure that the appearing confirmation dialogue, depending on the choice, allows the removal of the site or does nothing. 

## Reviewer Checklist:


Tests



Two translations added under the name: banSiteConfirmation. Mechanism for creating the confirmation dialogue mimics that shown in the syncTable component. Ref: [here](https://github.com/brave/browser-laptop/blob/master/app/renderer/components/preferences/syncTab.js#L392)
